### PR TITLE
Create data model

### PR DESCRIPTION
**Pull request by luis-pomare**
In this milestone, we have:
- Created the linter configuration to run locally and remotely.
- Created the database model.
- Created the association for the following ERD diagram:
ERD

### CODE REVIEW

**Review by GzimAsani — CHANGES_REQUESTED** [review]
Hi @luis-pomare , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional README. Please use [this template](https://github.com/microverseinc/readme-template)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    t.index ["AuthorId"], name: "index_posts_on_AuthorId"
+  end
+
+  create_table "users", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.string "Name"
+    t.string "Photo"
+    t.text "Bio"
+    t.integer "PostsCounter"
```
Comment (andersonlebon) [inline db/schema.rb:57]: ## [OPTIONAL]
- [ ] Kindly follow the name in convention in rails you should use `_` to separate words and in small letters

File: db/migrate/20230502195240_add_user_ref_to_comments.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```
Comment (andersonlebon) [inline db/migrate/20230502195240_add_user_ref_to_comments.rb:3]: - [ ] Kindly make sure the comment has a user using the `author_id` as reference and not `user_id`

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @luis-pomare 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @luis-pomare,

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**